source/Makefile.vars

```diff
@@ -6,34 +6,37 @@ CPLUSPLUS      = icpc
 CPLUSPLUS_MPI = mpiicpc
 #CPLUSPLUS_MPI = /public/intel2017/impi/2017.1.132/intel64/bin/mpiicpc
 
-LAPACK_DIR    = $(MKLROOT)
-#LAPACK_DIR = /public/intel2017/compilers_and_libraries_2017.1.132/linux/mkl
-#LAPACK_DIR = $(MKLROOT)
-#LAPACK_DIR    = /public/intel2017/mkl
-
-FFTW_DIR = /home/mohan/1_Software/impi_fftw-3.3.8
-#FFTW_DIR = /home/qianrui/intelcompile/impi_fftw
-#FFTW_DIR       = /public/udata/xiaohui/software/fftw2
-#FFTW_DIR       =/opt/fftw/3.3.6-p12/intel/2017.update4
-#FFTW_DIR      = /public/fftw-3.3.8
-
-BOOST_DIR = /home/mohan/1_Software/impi_boost-1.70.0
-#BOOST_DIR = /home/qianrui/intelcompile/impi_boost
-#BOOST_DIR      = /public/udata/xiaohui/software/boost_1_39_0
-#BOOST_DIR      = /opt/boost/1.64.0
-
-ELPA_DIR = /home/mohan/1_Software/impi_elpa-16.05.005
-#ELPA_DIR = /home/qianrui/intelcompile/impi_elpa
-#ELPA_DIR   = /public/udata/xiaohui/ELPA-2016.05.004
-#ELPA_DIR = /opt/elpa/intel_2017_update4
-
-CEREAL_DIR = /home/mohan/1_Software/ABACUS_Github/cereal/cereal/
-#CEREAL_DIR = /home/qianrui/intelcompile/cereal/
-
-#DeePKS
-LIBTORCH_DIR = /home/fortneu49/soft/libtorch
-LIBNPY_DIR = /home/fortneu49/soft/libnpy
-#\DeePKS
+#==========================
+# LIB and INCLUDE
+#==========================
+HONG_LAPACK     = -D__MKL -DMKL_ILP64
+LAPACK_DIR    	= $(MKLROOT)
+LAPACK_INCLUDE  = -I${LAPACK_DIR}/include
+LAPACK_LIB_DIR  = ${LAPACK_DIR}/lib/intel64
+LAPACK_LIB      = -L${LAPACK_LIB_DIR} -Wl,--start-group -lmkl_intel_lp64 -lmkl_intel_thread -lmkl_core -lmkl_scalapack_lp64 -lmkl_blacs_intelmpi_lp64 -Wl,--end-group -Wl,-rpath=${LAPACK_LIB_DIR}
+
+HONG_FFTW       = -D__FFTW3
+FFTW_DIR		= /home/liyuanbo/soft/fftw
+FFTW_INCLUDE    = -I${FFTW_DIR}/include
+FFTW_LIB_DIR    = ${FFTW_DIR}/lib
+FFTW_LIB        = -L${FFTW_LIB_DIR} -lfftw3 -Wl,-rpath=${FFTW_LIB_DIR}
+
+ELPA_DIR		= /home/liyuanbo/soft/elpa-2021.05.002/build
+ELPA_INCLUDE    = -I${ELPA_DIR}/include/elpa-2021.05.002
+ELPA_LIB_DIR	= ${ELPA_DIR}/lib
+ELPA_LIB     	= -L${ELPA_LIB_DIR} -lelpa -Wl,-rpath=${ELPA_LIB_DIR}
+
+#LIBXC_INCLUDE   = -I${LIBXC_DIR}/include
+#LIBXC_LIB_DIR   = ${LIBXC_DIR}/lib
+#LIBXC_LIB       = -L${LIBXC_LIB_DIR} -lxc -Wl,-rpath=${LIBXC_LIB_DIR}
+
+CEREAL_DIR 		= /home/liyuanbo/soft/cereal/
+CEREAL_INCLUDE 	= -I${CEREAL_DIR}/include
+
+#============DeePKS================
+#LIBTORCH_DIR = /home/fortneu49/soft/libtorch
+#LIBNPY_DIR = /home/fortneu49/soft/libnpy/include
 
 OBJ_DIR = obj
+OBJ_DIR_serial = obj_serial
```

Comment:
Why split serial into another `OBJ_DIR`?

Reply:
I didn't change this line, but I agree that this should not be seperated. I'll fix it.